### PR TITLE
changing readme formatting

### PR DESCRIPTION
**Pull request by nturgut**
changing readme formatting

### CODE REVIEW

File: packages/web_drivers/README.md [hunk 1]
```diff
@@ -1,23 +1,19 @@
-# To install and start Chrome Driver.
+**To install and start Chrome Driver.**
```
Comment (yjbanov) [inline packages/web_drivers/README.md]: Up to you, but why not keep it as normal text?

File: packages/web_drivers/README.md [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
 
-dart lib/web_driver_installer.dart chromedriver &
+`dart lib/web_driver_installer.dart chromedriver &`
```
Comment (yjbanov) [inline packages/web_drivers/README.md]: Everywhere else we use triple backticks for non-inline code snippets ([example](https://github.com/flutter/flutter/blob/master/dev/devicelab/README.md))

Comment (yjbanov) [reply packages/web_drivers/README.md]: BTW, do we need the `&` at the end? It prevents the ability to use Ctrl + C to shutdown the driver. I'm sure developers can consult with StackOverflow or what not to learn how to run a process in the background.

File: packages/web_drivers/README.md (hunk 1 above)
Comment (nturgut) [reply packages/web_drivers/README.md]: I like different shades. It makes easier for me concentrate while reading.

File: packages/web_drivers/README.md (hunk 2 above)
Comment (nturgut) [reply packages/web_drivers/README.md]: > Everywhere else we use triple backticks for non-inline code snippets ([example](https://github.com/flutter/flutter/blob/master/dev/devicelab/README.md))

good idea.

**Pull request merged**